README.md

```diff
@@ -21,7 +21,17 @@ a job manually. If Fedimg is installed properly, the `bin/trigger_upload.py`
 script can be used for this purpose:
 
 ```
-./bin/trigger_upload.py SOME_RAWXZ_URL COMPOSE_ID
+usage: trigger_upload.py [-h] -u URL -c COMPOSE_ID [-p PUSH_NOTIFICATIONS]
```

Comment:
this should be `[-p]` since there is no arg data that is provided

Reply:
Actually right now it expects arg data. I would update the script so that it expects no arg data